src/resource-metadata/common.js

```diff
@@ -0,0 +1,41 @@
+exports.schemaUrl = ""
+
+function extractArchitecture(architectures) {
+    const aws_arch = architectures?.[0]
+
+    return (aws_arch === 'x86_64')
+        ? 'amd64'
+        : (aws_arch === 'arm64')
+            ? 'arm64'
+            : ''
+}
+
+exports.extractArchitecture = extractArchitecture;
+
+function intAttr(key, value) {
+    return {
+        key: key,
+        value: {
+            intValue: value,
+        },
+    }
+}
+
+exports.intAttr = intAttr;
+
+function stringAttr(key, value) {
+    return {
+        key: key,
+        value: {
+            stringValue: value,
+        },
+    }
+}
+
+exports.stringAttr = stringAttr;
+
```

Comment:
In ESM with arrow function it would be simply

```suggestion
export const intAttr = (key, value) => ({
    key,
    value: {
        intValue: value,
    },
});

export const stringAttr = (key, value) => ({
    key,
    value: {
        stringValue: value,
    },
});
```
But whatever works for you guys.